timelock-impl/src/test/java/com/palantir/atlasdb/timelock/transaction/timestamp/DelegatingClientAwareManagedTimestampServiceTest.java

```diff
@@ -114,6 +114,37 @@ public void makesRequestsAgainIfTimestampRangeDoesNotIncludeCorrectModuli() {
 
         verify(allocator, times(3)).getRelevantModuli(UUID_TWO);
         verify(timestamps, times(3)).getFreshTimestamps(anyInt());
+    }
+
+    @Test
+    public void batchedTimestampCallShouldReturnAtLeastOneUsableTimestamp() {
+        when(allocator.getRelevantModuli(UUID_ONE)).thenReturn(RESIDUE_ONE);
+        when(timestamps.getFreshTimestamps(anyInt()))
+                .thenReturn(TIMESTAMP_RANGE);
+
+        assertThat(service.getFreshTimestampsForClient(UUID_ONE, 2).stream())
```

Comment:
is there a containsExactly method here? `assertThat(service.getFreshTimestampsForClient(UUID_ONE, 2).stream()).containsExactly(RESIDUE_ONE_TIMESTAMP_IN_RANGE.timstamp());`